dev/devicelab/lib/framework/utils.dart

```diff
@@ -585,6 +585,9 @@ void cd(dynamic directory) {
 
 Directory get flutterDirectory => Directory.current.parent.parent;
 
+// TODO (kaushik) : this is currently not discoverable, can we set an env var?
+Directory get openpayDirectory => Directory.current.parent.parent;
```

Comment:
Would it be possible to have an environment variable like `OPENPAY_CHECKOUT_PATH` be passed? Looking [here](https://flutter-review.googlesource.com/c/recipes/+/31583/4/recipes/devicelab/devicelab_drone.py#65) the checkout seems to be at a temp path.

Reply:
Support is here: https://flutter-review.googlesource.com/c/recipes/+/31940